js/stats.js

```diff
@@ -0,0 +1,60 @@
+$(function() {
+  if (!window.location.pathname.match(/\/about.$/)) {
+    return;
+  }
+  var hostname = $('html').data('api-hostname') || "";
+
+  $.ajax({
+    type: 'GET',
+    url: hostname + '/api/stats',
+  })
+  .done(function(resp) {
+    load_chart('top-million-chart', resp.response.TOP_DOMAINS, '#f938bd', function(item) {
+      return item.value + ' domains';
+    });
+    load_chart('visitors-chart', resp.response.LOCAL, '#20c9c9', function(item) {
```

Comment:
Is there a way to constant-ize these hex colors?

Reply:
There's not a way I'm super into to share them with the CSS but I can put them in js vars further up!

Reply:
It'd be cool if we could use the same variable names that are in variable.scss.